dev-support/bin/yetus-wrapper

```diff
@@ -77,11 +81,17 @@ HADOOP_YETUS_VERSION=${HADOOP_YETUS_VERSION:-0.8.0}
 BIN=$(yetus_abs "${BASH_SOURCE-$0}")
 BINDIR=$(dirname "${BIN}")
 
+## HADOOP_YETUS_VERSION >= 0.9.0 the tarball named with apache-yetus prefix
+if version_ge $HADOOP_YETUS_VERSION 0.9.0; then
```

Comment:
shellcheck:15: note: Double quote to prevent globbing and word splitting. [SC2086]